addons/l10n_ar/views/report_invoice.xml

```diff
@@ -1,43 +1,261 @@
+            <span groups="account.group_show_line_subtotals_tax_included">Amount</span>
+        </span>
+        <span t-field="line.price_subtotal" position="attributes">
+            <attribute name="t-field">line.l10n_latam_price_subtotal</attribute>
+        </span>
+        <!-- if b2c we still wants the latam subtotal -->
+        <span t-field="line.price_total" position="attributes">
+            <attribute name="t-field">line.l10n_latam_price_subtotal</attribute>
+        </span>
+        <span t-field="o.amount_untaxed" position="attributes">
+            <attribute name="t-field">o.l10n_latam_amount_untaxed</attribute>
+        </span>
+
+        <!-- use column vat instead of taxes and only if vat discriminated -->
+        <th name="th_taxes" position="replace">
+            <th t-if="o.amount_by_group" name="th_taxes" t-attf-class="text-left {{ 'd-md-table-cell' if report_type == 'html' else '' }}"><span>% VAT</span></th>
+        </th>
+        <!-- use column vat instead of taxes and only list vat taxes-->
+        <td t-attf-class="text-left {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}" position="replace">
+            <td t-if="o.amount_by_group" t-attf-class="text-left {{ 'd-md-table-cell' if report_type == 'html' else '' }}">
+                <span t-esc="', '.join(map(lambda x: (x.description or x.name), line.l10n_latam_tax_ids.filtered(lambda x: x.tax_group_id.l10n_ar_vat_afip_code)))" id="line_tax_ids"/>
+            </td>
+        </td>
+
+        <!-- remove payment term, this is added on information section -->
+        <p t-if="o.invoice_payment_term_id" position="replace"/>
+
+        <!-- remove payment reference that is not used in Argentina -->
+        <p t-if="o.type in ('out_invoice', 'in_refund')" position="replace"/>
+
+        <!-- replace information section and usage argentinian style -->
+        <div id="informations" position="replace">
+            <div id="informations" class="row mt8 mb8">
+                <div class="col-6">
+
+                    <!-- IDENTIFICACION (ADQUIRIENTE-LOCATARIO-PRESTARIO) -->
+
+                    <!-- (14) Apellido uy Nombre: Denominicacion o Razon Soclial -->
+                    <strong>Customer: </strong><span t-field="o.partner_id.name"/>
+
+                    <!-- (15) Domicilio Comercial -->
+                    <br/>
+                    <span t-field="o.partner_id" t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;], &quot;no_marker&quot;: true, &quot;no_tag_br&quot;: True}"/>
+
+                    <!-- (16) Responsabilidad AFIP -->
+                    <strong>VAT Cond: </strong><span t-field="o.partner_id.l10n_ar_afip_responsibility_type_id"/>
+
+                    <!-- (17) CUIT -->
+                    <t t-if="o.partner_id.vat and o.partner_id.l10n_latam_identification_type_id.name and o.partner_id.l10n_latam_identification_type_id.name != 'Sigd'">
```

Comment:
name !=....  (never rely on name)

Reply:
@jco-odoo 
Hi Jos, the requested change has been pushed.
Please let us know if there is another thing pending in order to merge this PR.

Best Regards

cc @jjscarafia